src/content/nft/index.md

```diff
@@ -305,9 +304,9 @@ Decentralized meaning you and everyone else can verify you own something. All wi
 
 Secure meaning no one can copy/paste your NFT or steal it.
 
-These qualities of Ethereum makes digitally owning unique items and getting a fair price for your content possible. But it comes at a cost. Blockchains like Bitcoin and Ethereum are energy intensive right now because it takes a lot of energy to preserve these qualities. If it was easy to rewrite Ethereum's history to steal NFTs or cryptocurrency, the system collapses.
+These qualities of Ethereum makes digitally owning unique items and getting a fair price for your content possible. Ethereum protects the assets using a decentralized consensus mechanism called ['proof-of-stake'](/developers/docs/consensus-mechanisms/pos). This is a low carbon consensus mechanism that is also considered more secure than the energy-intensive alternative, ['proof-of-work'](/developers/docs/consensus-mechanisms/pow). NFTs have been associated with high energy expenditure because Ethereum used to be secured using proof-of-work. This is no longer true.
```

Comment:
If possible, I think we should try to avoid strictly calling proof-of-stake a consensus mechanism as it is just one piece of consensus to determine who is a valid block producer.
```suggestion
These qualities of Ethereum makes digitally owning unique items and getting a fair price for your content possible. Ethereum protects the assets using a decentralized consensus mechanism which involves ['proof-of-stake'](/developers/docs/consensus-mechanisms/pos). This is a low carbon method to determine who can add a block of transactions to the chain, and is considered more secure than the energy-intensive alternative, ['proof-of-work'](/developers/docs/consensus-mechanisms/pow). NFTs have been associated with high energy expenditure because Ethereum used to be secured using proof-of-work. This is no longer true.
```